src/libraries/Directory.Build.props

```diff
@@ -1,7 +1,7 @@
 <!-- Contains common properties that apply to projects under the Libraries folder -->
 <Project>
   <PropertyGroup>
-    <LangVersion>9.0</LangVersion>
+    <LangVersion>10.0</LangVersion>
```

Comment:
why this change?

Reply:
I made a comment in the other file
this is to allow "global usings" which is a C# 10 feature
[https://devblogs.microsoft.com/dotnet/welcome-to-csharp-10/](https://devblogs.microsoft.com/dotnet/welcome-to-csharp-10/)

This is used to facilitate he use of shims in .Net 4.6.2 test projects